import-automation/executor/app/main.py

```diff
@@ -103,7 +103,6 @@ def scheduled_updates():
             repo_name=config.github_repo_name,
             auth_username=config.github_auth_username,
             auth_access_token=config.github_auth_access_token),
-        dashboard=dashboard_api.DashboardAPI(config.dashboard_oauth_client_id),
```

Comment:
Similar to #836, the dashboard is simply not running. We need to disable the dashboard to get the executor to run. Will leave that as a separate problem.